core/src/main/java/io/github/wysohn/triggerreactor/core/script/parser/Parser.java

```diff
@@ -1171,6 +1170,30 @@ private Node parseId(Deque<Node> deque) {
         return stack.pop();
     }
 
+    private Node parseImport(Token importToken) throws IOException, ParserException {
+        Node node = new Node(importToken);
+
+        if (token.type == Type.ENDL) {
+            return node;
+        }
+
+        String attribute = (String) token.value;
+        if ("AS".equalsIgnoreCase(attribute)) {
+            nextToken();
+
+            if (token == null || token.type != Type.ID || ((String) token.value).charAt(0) == '#') {
```

Comment:
just out of curiosity, what is the pound token (#) check for? in case the Executor expression is used?

Reply:
Yes, `import.as` token registers next token to the importMap. But if it starts with pound token (#), lexer will understood that as Executors.

Reply:
Don't pay attention to the comment above.
It works even start with pound token (#), but I think to check just in case as the convention.

Reply:
> Don't pay attention to the comment above. It works even start with pound token (#), but I think to check just in case as the convention.

Yeah I was just wondering cause if we would check for executors explicitly, then why not for the placeholder, which works the same as executors xD

I didn't manage to look really carefully into it, so I decided to ask you instead :]

Reply:
> Yeah I was just wondering cause if we would check for executors explicitly, then why not for the placeholder, which works the same as executors xD
> 
> I didn't manage to look really carefully into it, so I decided to ask you instead :]

The token type of Executor and IMPORT matches with `Type.ID`, but the placeholder naturally causes an error because it's token type is Operator.

Reply:
oh placeholder is an operator? that's strange cause it would semantically make sense to have them as the same type.

Reply:
Clearly, Lexer tokenize "$playername" as $ to `Type.OPERATOR` and "playername" to `Type.ID`.

Reply:
> Clearly, Lexer tokenize "$playername" as $ to `Type.OPERATOR` and "playername" to `Type.ID`.

Now that reminds me that placeholder can take in the arguments, so I think that was why it became an operator.